src/network/udp.py

```diff
@@ -97,22 +97,30 @@ def bm_command_addr(self):
             time.time()
         return True
 
+    # override the protocol do ignore the following commands on UDP
+    # we do this mainly for security reasons
```

Comment:
This is more of inheritance flow. Maybe `UDPSocket` should inherit from some more basic class?

Reply:
Perhaps.